priv/ie_gen_v1.erl

```diff
@@ -1045,7 +1045,9 @@ raw_ies() ->
       [{"Line", 32, integer},
        {'_', 0}]},
      {{18681, 10}, "TP Created NAT Binding",
-      [{"Group", 0, {type, v1_grouped}}]}
+      [{"Group", 0, {type, v1_grouped}}]},
+     {{18681, 11}, "TP IPFIX Policy",
+      [{"Policy", 8, length_binary}]}
```

Comment:
that does not match the definition of the IPFIX IE in UPF @ https://github.com/travelping/upg-vpp/pull/219/files#diff-7d05fe23942d2891d030ef3ec5582c6bc23c466d8dc84c94ba01c7dfd5f60ddaR4758
```suggestion
      [{"Policy", 0, binary}]}
```